partner_document/i18n/es.po

```diff
@@ -40,6 +40,7 @@ msgstr "Clasificación"
 
```

Comment:
Use always ca.po instead of ca_ES.po

The most used is ca in OCA translations, check [this](https://translation.odoo-community.org/languages/) out

<img width="1392" height="389" alt="image" src="https://github.com/user-attachments/assets/1d552d0c-62ca-4fd7-9bda-8d8217d757ba" />

And in Odoo they don't even use ca_ES at all